install/jsp/bin/sig-test-pkg-list_se8.txt

```diff
@@ -22,15 +23,6 @@
 # parent package's signature is checked.
 ##
 
-java.io
-java.lang
-java.util
-jakarta.el
-jakarta.servlet
 jakarta.servlet.jsp
 jakarta.servlet.jsp.el
-jakarta.servlet.jsp.jstl.core
```

Comment:
Is it required to remove these. We add the corresponding jars of these packages in the sigTestClasspath while generating the signature file and also to run the tests.

Reply:
As far as I could tell, yes. The Java 11 signature test machinery doesn't provide an easy way to exclude packages from the signature tests so you need to make sure they aren't present. There might be a better way of achieving the same result but I couldn't find one.

Reply:
For java 11,  the corresponding file is install/jsp/bin/sig-test-pkg-list_se11.txt. Can you make these changes in this file.

Reply:
I was thinking since sig-test-pkg-list_se8.txt is used for jdk8 runs which should work , was not sure why this is changed. But the packages removed are not required to be tested for jsp standalone I presume.

Reply:
Correct. Those packages aren't part of the JSP spec. It did work for Java 8 both with and with out those changes. I was trying to avoid future confusion if someone compared the Java 8 signatures with those for Java 11 and wondered why they were wildly different.